src/map/constants.inc.h

```diff
@@ -0,0 +1,970 @@
+	script->constdb_comment("Status Icons");
```

Comment:
I mean you introduced file contants.inc.h. It not exists in hercules before your changes.
Probably better add this file in separate commit before any clan related changes.
Also in this file code not for headers. Not sure may be better name it like constants.inc, or constants.inc.c
because from code inside this is not header.